inspirehep/modules/theme/templates/inspirehep_theme/format/record/Inspire_Default_HTML_brief.tpl

```diff
@@ -37,7 +37,7 @@
               </div>
               <div class="col-md-11">
                 <h4 class="custom-h">
-                    <a class="title" href="{{ url_for('invenio_records_ui.recid', pid_value=record['control_number']) }}">
+                    <a class="title" href="{{ url_for('invenio_records_ui.literature', pid_value=record['control_number']) }}">
```

Comment:
This will eventually cause troubles for other collections but we can add that later.